### DISCUSSION TITLE
Flexmock development/Maintenance

### DISCUSSION THREAD

**Discussion by navkore** [flexmock/flexmock#147] [General]
Hi all,
I  have used Flexmock for a long time now. But I see that there have been no releases after 2022. Is the development of this nice mock tool stopped? Just asking as my team doesn't want to use a tool that is not maintained further.  

As I loved working with Flexmock I thought I would just ask you once before going forward.

Thanks!
Naveen

**Comment by ollipa** [comment]
Hi @navkore, the fact that there has not been a release for a while is just because there was no need for that. Most of the changes lately have been just housekeeping and internal changes. We are not currently actively developing any new features, but we will keep the library updated if any serious bugs or compatibility issues would arise.

Is there anything in particular that you are missing?
